ui/app/src/Alerts/Api.elm

```diff
@@ -27,10 +27,10 @@ fetchAlertGroups apiUrl filter =
     Utils.Api.send (Utils.Api.get url (Json.Decode.list Data.AlertGroup.decoder))
 
 
-fetchAlerts : String -> Filter -> Cmd (ApiData (List GettableAlert))
+fetchAlerts : String -> Filter -> Cmd (ApiData (List ExtendedGettableAlert))
 fetchAlerts apiUrl filter =
     let
         url =
             String.join "/" [ apiUrl, "alerts" ++ generateAPIQueryString filter ]
     in
-    Utils.Api.send (Utils.Api.get url (Json.Decode.list Data.GettableAlert.decoder))
+    Utils.Api.send (Utils.Api.get url (Json.Decode.list Data.ExtendedGettableAlert.decoder))
```

Comment:
This is the most important change to the Elm code. I'm decoding the response for `/api/v2/alerts` as `ExtendedGettableAlert` when in the OpenAPI schema it is `GettableAlert`. The difference between the two is that `ExtendedGettableAlert` has a `mutedBy` column in `status`.